src/css/popup.css

```diff
@@ -687,7 +687,132 @@ sign-up-panel::after {
   gap: var(--spacingSm);
   margin-bottom: var(--spacingSm);
 }
+.fx-relay-csat-survey-link-container {
+  background-color: var(--colorViolet60);
+  border-radius: var(--borderRadiusSm);
+  padding: var(--spacingSm);
+  color: var(--colorWhite);
+  font-size: var(--fontSizeBodyXs);
+  display: flex;
+  justify-content: center;
+  gap: var(--spacingSm);
+  margin-bottom: var(--spacingSm);
+  align-items: center;
+}
+/* TODO: down to 730 - reuse */
+
+.fx-relay-csat-survey-link {
+  color: var(--colorWhite);
+  text-decoration: underline;
+  font-size: var(--fontSizeTitle3xs);
+  padding: var( --spacingXs) var(--spacingMd);
+  cursor: pointer;
+  font-weight: 600;
+  text-underline-offset: var(--spacingXs);
+}
+
+.fx-relay-csat-survey-link,
+.fx-relay-csat-survey-close-icon {
+  margin-left: auto;
+}
+
+.fx-relay-csat-survey-close-icon {
+  padding: 0 var(--spacingXs);
+  cursor: pointer;
+}
+
+.csat-survey-satisfaction-section {
+  margin-bottom: var(--spacingMd);
+  display: flex;
+  gap: 6px;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
 
+.csat-survey-text {
+  margin: 0 auto;
+  color: var(--colorGrey50)
+}
+
+#survey-panel .fx-relay-panel-content {
+  min-height: 344px;
+  max-height: 344px; 
```

Comment:
Note for any other reviewers: This magic number follows a known issue/pattern across the popup panel(s). 😅